setup.bat

```diff
@@ -0,0 +1,46 @@
+@echo off
+REM Check for Python version and set up a virtual environment
+
+REM Check for Python existence
+python --version >nul 2>&1
+if errorlevel 1 (
+    echo Error: Python is not installed. Please install Python.
+    exit /b 1
+)
+
+REM Get Python version
+for /f "delims=" %%i in ('python -c "import platform; print(platform.python_version())"') do set PYTHON_VERSION=%%i
+
+REM Check Python version
+echo Found Python version %PYTHON_VERSION%
+if "%PYTHON_VERSION%" lss "3.8" (
```

Comment:
I just tested this with Python version 3.11.6 and it printed the error message:

Found Python version 3.11.6
Error: Python version is too old. Please install Python between 3.8 and 3.11

Reply:
@aarondr77 wanna just remove this check? You can do it manually -- it's kinda overkill.